lib/registries/addon/drafts/draft/page/template.hbs

```diff
@@ -25,9 +25,19 @@
             {{compute draftManager.onPageChange this.pageIndex this.inReview}}
             <OsfLayout @backgroundClass={{local-class 'ContentBackground'}} as |layout|>
                 <layout.heading local-class='Heading'>
-                    <h1 local-class='Title'>
-                        {{t 'registries.drafts.draft.form.new_registration'}}
-                    </h1>
+                    <div local-class='Title'>
```

Comment:
can this be done without the extra `div` thrown in here? seems like we might be able to consolidate some of the styles on `.Heading` and `.Title`, especially since the heading only has one child

Reply:
I've poked around at it a little bit to see if it's possible to get rid of this extra wrapper div, but it is there to match the margin with the alignment of the side-nav icons (`max-width` + `margin: auto`). Without it, it's a little tricky for me to get them to align, unless there is an alternative that I'm not thinking of.

Reply:
> No.

Yuhuai (circa. 2019)

Reply:
ok, i will trust